### ISSUE TITLE
Can I disable text validation?

### ISSUE THREAD

**Issue by stelmath** [proycon/foliapy#26]
Hello and thanks for the repo,

I think I have an issue with text validation. I have created a folia Doc with sentence elements and their text. I am machine translating it and then reading the machine translated xml as a new folia Document. No issue so far, then I am creating a new folia Document that has div elements and inside it I am appending sentence pairs, one from each of those two folia Documents (the original one and the one that I machine translated). But, when I try to append this sentence from the machine translated Document I get: 

```
folia.main.InconsistentText: Text for <Sentence at 140523773674208 id=source_segment_EN_7 set=None class=None>, is inconsistent: EXPECTED (deep text after normalization) *****>
[ date ]
****> BUT FOUND (strict text after normalization) ****>
[date]
******* DEVIATION POINT: [<*HERE*>date]
(also checked against older rules prior to FoLiA v2.4.1)
```

Basically my sentence is : 
```
<s xml:id="segment_id_7">
      <t>
        <t-style class="bold">[date]</t-style>
      </t>
</s>
```
which results in the above error. I don't understand what the term deep text and strict text means but can I simply turn off this validation and let this sentence be appended in a div? Thanks

**Comment by proycon** [comment]
Text validation is a rather essential component that ensures your FoLiA document is valid, so it always enabled. 

I wonder what causes your error, do you also have a `<t>` element under your
`<div>`? The error suggests that the text there is `[ date ]` (with spaces). If
text is specified multiple times on multiple levels (text redundancy), it can't
be different on a deeper level than on the higher level; text validation

**Comment by stelmath** [comment]
To make it simpler, I have isolated the sentence that causes the issue. Here is a `debug.xml` document:

```
<?xml version='1.0' encoding='utf-8'?>
<FoLiA xmlns="http://ilk.uvt.nl/folia" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="task_id_1" version="2.5.1" generator="foliapy-v2.5.8">
  <metadata type="native">
    <annotations>
      <text-annotation set="https://raw.githubusercontent.com/proycon/folia/master/setdefinitions/text.foliaset.ttl"/>
      <pos-annotation set="https://web.archive.org/web/20190206204307/https://www.clips.uantwerpen.be/pages/mbsp-tags">
        <annotator processor="proc.spacy.d01e0445"/>
      </pos-annotation>
      <style-annotation set="https://raw.githubusercontent.com/proycon/folia/master/setdefinitions/styles.foliaset.xml">
        <annotator processor="proc.internal_formatting.452d069a"/>
      </style-annotation>
      <spanrelation-annotation>
        <annotator processor="proc.word_alignment.2ff3e2c3"/>
      </spanrelation-annotation>
      <sentence-annotation/>
      <token-annotation/>
    </annotations>
    <provenance>
      <processor xml:id="proc.spacy.d01e0445" name="spacy" type="auto">
        <processor xml:id="proc.spacy.d01e0445.generator" name="foliapy" type="generator" version="2.5.8" folia_version="2.5.1"/>
      </processor>
      <processor xml:id="proc.internal_formatting.452d069a" name="internal_formatting" type="auto">
        <processor xml:id="proc.internal_formatting.452d069a.generator" name="foliapy" type="generator" version="2.5.8" folia_version="2.5.1"/>
      </processor>
      <processor xml:id="proc.word_alignment.2ff3e2c3" name="word_alignment" type="auto">
        <processor xml:id="proc.word_alignment.2ff3e2c3.generator" name="foliapy" type="generator" version="2.5.8" folia_version="2.5.1"/>
      </processor>
    </provenance>
  </metadata>
  <text xml:id="task_id_1.text.1">
  <s xml:id="segment_id_7">
      <t>
        <t-style class="bold">[date]</t-style>
      </t>
      <w xml:id="segment_id_7.w.1">
        <t>[</t>
        <pos class="XX"/>
      </w>
      <w xml:id="segment_id_7.w.2">
        <t>date</t>
        <pos class="XX"/>
      </w>
      <w xml:id="segment_id_7.w.3">
        <t>]</t>
        <pos class="XX"/>
      </w>
    </s>
  </text>
</FoLiA>
```

```
import folia.main as folia

folia = folia.Document(file='debug.xml')
```

**Comment by kosloot** [comment]
There are 2 possible solutions here:
1.  fix the words, by adding the ``space="no"`` atribute  for words 1 and 2.
2. fix the sentence by adding spaces after the ``[`` and after ``date``

So either:
```xml
      <w xml:id="segment_id_7.w.2" space="no">
        <t>date</t>
        <pos class="XX"/>
      </w>
      <w xml:id="segment_id_7.w.3">
        <t>]</t>
        <pos class="XX"/>
      </w>
```

or:
```xml
      <t>
        <t-style class="bold">[ date ]</t-style>
      </t>
```

The point is, that spaces are implicit after EVERY word
